backend/internal/common/env_config.go

```diff
@@ -76,4 +90,35 @@ func init() {
 	if parsedAppUrl.Path != "" {
 		log.Fatal("PUBLIC_APP_URL must not contain a path")
 	}
+
+	validTraceExporters := []string{"http", "grpc", "stdout"}
+	validMetricsExporters := []string{"http", "grpc", "stdout", "prometheus"}
+
+	validateExporters(EnvConfig.TracesExporters, validTraceExporters, "TRACES_EXPORTERS")
+	validateExporters(EnvConfig.MetricsExporters, validMetricsExporters, "METRICS_EXPORTERS")
+}
+
+func validateExporters(exps []string, valid []string, env string) {
+	if len(exps) != 0 {
+		if EnvConfig.AppEnv == "production" && slices.Contains(exps, "stdout") {
+			log.Fatal("stdout exporters must not be used in production")
```

Comment:
Is this really necessary? While I agree it is not a good idea to do this, why crashing the app?

Reply:
It's not necessary, but it avoids people running with bad configurations.

The output formats of the stdout exporters are not stable, and as such can never be relied on. They really should not be enabled unless you know what you're doing, and leaving them accidentally enabled is far from ideal.

Reply:
This is gone now since everything is configured by the OTEL variables. Since this package doesn't own them I'm no longer doing any validation.